doc/source/index.rst

```diff
@@ -114,29 +114,64 @@ https://github.com/Billingegroup/scikit-package/graphs/contributors.
 Acknowledgements
 ================
 
-The Billinge Group's scikit-package has been modified from the NSLS-II scientific cookiecutter: https://github.com/nsls-ii/scientific-python-cookiecutter
+The Billinge Group's ``scikit-package`` has been modified from the NSLS-II scientific cookiecutter: https://github.com/nsls-ii/scientific-python-cookiecutter
+
+
+.. toctree::
```

Comment:
modeling off of https://github.com/readthedocs/readthedocs.org/blob/main/docs/user/index.rst